src/Stripe.net/Infrastructure/Public/Mapper.cs

```diff
@@ -5,9 +5,14 @@ namespace Stripe
     using System.Reflection;
     using Newtonsoft.Json;
     using Newtonsoft.Json.Linq;
+    using Stripe.Infrastructure;
 
     public static class Mapper<T>
     {
+        private static JsonConverter[] converters = {
+            new ExternalAccountConverter(),
```

Comment:
so this is the new place where we would list all converters?

Reply:
Yep. Not sure if this is the best way to achieve this, but it seems to work and should be relatively efficient (it's `static`, so only initialized once).